_source/logzio_collections/_log-sources/elastic-container-service.md

```diff
@@ -60,15 +62,13 @@ by setting the `ignoreOlder` parameter of the new docker
 to a lower value (for example, `20m`).
 {:.info-box.note}
 
-| Parameter | Description |
+| Paramater | Details |
 |---|---|
-| LOGZIO_TOKEN <span class="required-param"></span> | Your Logz.io account token. {% include log-shipping/replace-vars.html token=true %} |
-| LOGZIO_URL <span class="required-param"></span> | Your Logz.io listener URL and port. {% include log-shipping/replace-vars.html listener=true %} |
- ignoreOlder <span class="default-param">`3h`</span>|  Set a time limit on back shipping logs. Upgrading to a newer version of docker-collector-logs while it is already running will cause it to resend logs that are within the `ignoreOlder` timeframe. You can minimize log duplicates by setting the `ignoreOlder` parameter of the new docker to a lower value (for example, `20m`). |
-{:.paramlist}
+| **LOGZIO_URL** | **Required**. Your Logz.io listener URL. Replace `<LISTENER-URL>` with your region's listener URL. For more information on finding your account's region, see [Account region](https://docs.logz.io/user-guide/accounts/account-region.html) in the Logz.io Docs. |
```

Comment:
@shalper If this PR is indeed going through, these table rows need to match the docs format - with the 'default' and 'required' `<span>` tags, not-bolded parameter name, and using the Jekyll include.